Jenkinsfile

```diff
@@ -459,7 +459,11 @@ pipeline {
                         STACKTRACE_OPTION="--stacktrace"
                     fi
 
-                    /opt/ort/bin/ort $LOG_LEVEL $STACKTRACE_OPTION report -f CycloneDX,NoticeByPackage,NoticeSummary,SpdxDocument,StaticHTML,WebApp -i out/results/current-result.yml -o out/results/reporter
+                    /opt/ort/bin/ort $LOG_LEVEL $STACKTRACE_OPTION report \
+                        -f CycloneDX,NoticeTemplate,SpdxDocument,StaticHTML,WebApp \
+                        -i out/results/current-result.yml \
+                        -o out/results/reporter \
+                        -O NoticeTemplate=template.id=default,summary
```

Comment:
Nit: Maybe put reporter-specific options closer to the formats definitions, i.e. in line 464, after the `-f` line?

Reply:
Done.